doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -334,8 +334,7 @@ julia> typeof(x)
 Float64
 ```
 
-Half-precision floating-point numbers are also supported ([`Float16`](@ref)), but they are
-implemented in software and use [`Float32`](@ref) for calculations.
+Half-precision floating-point numbers are also supported ([`Float16`](@ref)) on all platforms, with native instructions used on hardware which supports this number format, otherwise operations are implemented in software and use [`Float32`](@ref) for intermediate calculations.
```

Comment:
Maybe point out that this equivalent to gcc `-fexcess-precision=ieee`

Reply:
I presume you meant [`-fexcess-precision=standard`](https://gcc.gnu.org/onlinedocs/gcc/Half-Precision.html)?

Reply:
I think we actually implement `-fexcess-precision=16` https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html#index-fexcess-precision

Reply:
We techically just do the LLVM default nowadays, the demote float16 pass is basically unused since https://github.com/JuliaLang/julia/issues/55486

Reply:
Ok, I tried to incorporate both comments above, explaining this is an implementation detail (but I think some non-Julia people reading the manual may be interested in knowing it)